docs/data/joy/components/modal/modal.md

```diff
@@ -75,14 +75,30 @@ The possible values are:
 
 {{"demo": "CloseModal.js"}}
 
-### Modal Dialog
+### Modal dialog
 
 To create a modal dialog, renders the `ModalDialog` component inside the `Modal`.
```

Comment:
```suggestion
To create a modal dialog, render the Modal Dialog component inside the Modal.
```

Reply:
This whole page needs to be revised to match current style guidelines (Modal Dialog instead of `ModalDialog`) but we can start with this new section.